lib/codelog/command/new.rb

```diff
@@ -5,18 +5,19 @@ module Command
     class New
       include FileUtils
 
-      def self.run
-        Codelog::Command::New.new.run
+      def self.run(options)
+        Codelog::Command::New.new.run options
       end
 
-      def run
+      def run(options)
         chdir Dir.pwd do
           # This script create a change file for the changelog documentation.
 
           full_file_name = "changelogs/unreleased/#{Time.now.strftime('%Y%m%d%H%M%S%L')}_change.yml"
 
           puts "== Creating #{full_file_name} change file based on example =="
           system! "cp changelogs/template.yml #{full_file_name}"
+          system! "${VISUAL:-${EDITOR:-nano}} #{full_file_name}" if options[:edit]
```

Comment:
@luisbevilacqua For me, this command is a bit obscure. Could you add a comment about it?